.gitignore

```diff
@@ -28,3 +28,4 @@ package-lock.json
 !/standalone/**/package-lock.json
 packages/web-component-library-angular/src/directives/proxies.ts
 packages/web-component-library-angular/src/directives/angular-component-lib/utils.ts
+package-lock.json
```

Comment:
@Ollie-nl Ik besef dat het erg irritant is dat `package-lock.json` wijzigingen er telkens bij komen te staan, maar dit (belangrijke) bestand bij `.gitignore` zetten is denk ik niet de oplossing.